### ISSUE TITLE
Use BackdropCMS.org to store searchable meeting agendas linked to video recordings

### ISSUE THREAD

**Issue by docwilmot** [backdrop-ops/backdropcms.org#457]
I was wondering if an issue was discussed at a Thursday meeting, and realised it might have been, but finding out was going to be a hard job. 

We should use the website and a dedicated node type for the agenda, allowing registered users to edit there instead, and then at the end of the meeting link the video recording to that agenda item. 

Furthermore we should make brief notes about decisions made during the meeting there.

Then make it all Views-listed and searchable.

**Comment by klonos** [comment]
Keeping such meeting minutes would indeed help, but it will be some work to keep them up to date. I wonder how much of it could be automated, or at the very least delegated among numerous people so that this does not become a "chore" of one single person or two. I like how for example we have separate people for the different tasks for each release.

Besides having nodes for past meetings, it would also be useful to have reserved nodes for future meetings. I expect these to serve as go-to points to assist with questions/requests that are often asked in the Gitter channel such as "Is there going to be a meeting today?" or "Can issue xyz be added to the agenda?" (not everyone has access to the google docs, but we can allow people to link to tickets on GitHub), "Was the meeting recorded?" (sometimes google is acting up and recording the session is not possible) etc.

**Comment by Graham-72** [comment]
I don't think we need 'minutes' of meetings - I agree that this could be quite a bit of extra work.

But it would be helpful to have something like an annotated agenda which lists the numbers of all issues referenced (as usually already listed in the agenda), perhaps with an R (for referenced), and a D (for decision made). Hopefully for any decision made there would be a note added to the issue's comments and nothing more need be written.

**Comment by docwilmot** [comment]
Ha, I had this open just about to type the same. The videos are already the minutes. Just need the agenda searchable and linked to the video. And some way to note if a (final) decision was made would be nice.

**Comment by docwilmot** [comment]
Still a good idea, if I says so.
